File: javascript-closure-angular-client/API/Client/ReceiptTransactionIn.js
goog.provide('API.Client.ReceiptTransactionIn');

/**
 * @record
 */
API.Client.ReceiptTransactionIn = function() {}

/**
 * @type {!API.Client.PayRecHeader}
 * @export
 */
API.Client.ReceiptTransactionIn.prototype.header;

/**
 * @type {!Array<!API.Client.PaymentLinesIn>}
 * @export
 */
API.Client.ReceiptTransactionIn.prototype.lines;

File: dot-net-client/src/Avalara.AvaTax.Brazil.RestClient/Model/ReceiptTransactionIn.cs
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace Avalara.AvaTax.Brazil.RestClient.Model
{
    /// <summary>
    /// ReceiptTransactionIn
    /// </summary>
    [DataContract]
    public class ReceiptTransactionIn
    {
        /// <summary>
        /// Gets or Sets Header
        /// </summary>
        [DataMember(Name = "header", EmitDefaultValue = false)]
        public PayRecHeader Header { get; set; }

        /// <summary>
        /// Gets or Sets Lines
        /// </summary>
        [DataMember(Name = "lines", EmitDefaultValue = false)]
        public List<PaymentLinesIn> Lines { get; set; }
    }
}
